### PR TITLE
Hello World Microverse Pull Request 

### PR DESCRIPTION
**Pull request by BranBayou**
This is Hello World Microverse project.

### CODE REVIEW

**Review by nahnahdev — CHANGES_REQUESTED** [review]
Hi @BranBayou , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
:heavy_check_mark: Good commit messages
:heavy_check_mark: Good use of GitHub flow

## Required Changes ♻️
- [ ] I noticed that you are missing the  [Microverse Readme template](https://github.com/microverseinc/readme-template). This makes your project incomplete. Every section of the readme template is required with the exception of the FAQ section which is optional. So please make sure to add the README template to your project and fill in all the necessary contents :+1:
- [ ] I noticed that you have a README file in your main branch but since this is the branch you're submitting for review, please ensure that you have it in this branch as well. In that README I noticed that you excluded some of the sections. But as I said earlier, every section of the Microverse README template is compulsory so kindly include all the missing sections in the one you'll add to this branch and resubmit for review :wink: 
- [ ] I also noticed that the license link in the README file in your main branch is broken and leads to an error page. This is because you don't have a license file in your project. . 
- [ ] The MIT License is a popular open-source license that allows others to use, modify, and distribute your code as long as they give credit to you the original author, and include the original license text. You can use this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to learn how to add one to your project :wink: 
- [ ] Kindly ensure that the link to the license file that you will add to your README file will take you to the created license and not to an error page :wink: 
- [ ] Your `stylelint` linter checks are failing kindly fix this as well to adhere to all of CSS best practices :100: 
- [ ] Your pull request description is currently short and doesn't highlight the changes that were introduced in the branch. Kindly include a summary describing the introduced changes in more detail. 
## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,153 @@
...
+</details>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Key Features
+
+- npm packages
+- html, renders the project
+
```
Comment (souad988) [inline README.md]: this section is to describe the features built in the project, in this case it displays a basic hello world styled message.

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @BranBayou , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

while you have done a great work implementing all project requirements you missed one thing which is required to get this project approved, it's using  a correct github workflow which means that all project requirements should be implemented  in the feature branch, setting up linters is in the list of this project requirement but I don't see linter files here, would you please remove the work from main branch then setup linters again from this branch and submit the same PR for review .
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joel-kalema — APPROVED** [review]
Hi @BranBayou 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

The project is running successfully 👍 
 
## Optional suggestions

- [ ] [OPTIONAL] I can see you deleted the linters, kindly make sure you installed liters, check the [link](https://github.com/microverseinc/linters-config/tree/master/html-css) to see how to install 👍 

liters


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**